Microsoft.Toolkit.HighPerformance/Helpers/ObjectMarshal.cs

```diff
@@ -7,12 +7,12 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-namespace Microsoft.Toolkit.HighPerformance.Extensions
+namespace Microsoft.Toolkit.HighPerformance.Helpers
```

Comment:
Is this going to change again with #3743, should we put it in final spot now?

Reply:
It shouldn't, no. From what we decided in our previous conversation in #3422, we're moving all the extensions in the main namespace of each package, but we haven't discussed helpers yet, so they're all going to remain in the `*.Helpers` namespaces in #3743, at least for now 🙂